### PR TITLE
Portfolio work section

### PR DESCRIPTION
**Pull request by brainconnect93**
1. checkout to a new branch (portfolio-work-section)
2. type my work section HTML code and give it a befitting styling
3. type my about section HTML code and give it a befitting styling
4. rename some of the images as requested by linters
5. commit my work and push it to the branch

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+<section class="about-section">
+    <div class="about-container">
+        <div class="about-portfolio">
+            <div class="about-box">
+                <h2 class="about-title">About <br> Myself</h2>
+                <p class="about-note">Hello I'm a software developer! I can help you build a product , feature or website Look through
+                    some of my work and experience! If you like what you see and have a project you need to work on,
+                    don't hestiate to contact me.</p>
+                <div class="about-head">
+                    <p>let's connect</p>
```
Comment (omar-labana) [inline index.html]: - [x] Let's update this `p` tag to be an `hx` tag, the purpose of the text included is a call for action more than being a paragraph text.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+                        </ul>
+                    </div>
+                    <button type="button" class="see-project">See Project</button>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!-- MY ABOUT SECTION -->
+<section class="about-section">
```
Comment (omar-labana) [inline index.html:186]: - [x] This section has a fixed padding of `5.5rem` causing it break on small phone screens. kindly fix it so it's identical to the design.
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+                            <li> <div class="tag1">css</div></li>
+                            <li> <div class="tag2">javascript</div></li>
+                        </ul>
+                    </div>
+                    <button type="button" class="see-project">See Project</button>
+                </div>
+            </div>
+
+            <!-- 4TH WORK SECTION -->
+            <div class="work-box1">
```
Comment (omar-labana) [inline index.html:154]: - [x] Projects are kinda broken on the smallest mobile screen `320px`, there seems to be a fixed size for some elements, kindly fix it.
image

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @brainconnect93, 

Excellent work so far! 👌
You have implemented almost all the required features without any errors or bugs. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
_N/A_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @brainconnect93👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OLIPLICHE — COMMENTED** [review]
Hi TEAM, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that contains an old Pull Request. Please Create a new Pull Request for a fresh review of this milestone

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**